Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -1015,7 +1014,7 @@ private void createDefaultMotionObjects() {
     public static double DEFAULT_MARKER_SIZE = 5;
```

Comment:
This is confusing since the string value is "10". I would change it so the default value is defined/specified ONLY once in a single variable and then convert it to a string value for presentation.

Reply:
As explained earlier this is just an initial value that gets read from preference so it means nothing but I went ahead and changed it to 10 so it appears consistent.